starknet_py/net/models/chains.py

```diff
@@ -27,3 +28,16 @@ def chain_from_network(
         raise ValueError("Chain is required when not using predefined networks.")
 
     return chain
+
+
+def default_token_address_for_chain(chain_id: Optional[StarknetChainId] = None) -> str:
+    if chain_id not in [
+        StarknetChainId.TESTNET,
+        StarknetChainId.TESTNET2,
+        StarknetChainId.MAINNET,
+    ]:
+        raise ValueError(
```

Comment:
[nitpick]: The error message here might depend on the function usage context, so i'd avoid throwing in those kind of methods. I guess for the use-case is fine, just noting here.